cms/djangoapps/contentstore/tasks.py

```diff
@@ -517,13 +520,13 @@ def get_dir_for_filename(directory, filename):
         dirpath = get_dir_for_filename(course_dir, root_name)
         if not dirpath:
             with translation_language(language):
+                LOGGER.error('Course import %s: Could not find the {0} file in the package.', courselike_key)
```

Comment:
fix: `{0}`, you are only passing `courselike_key` as an argument but the string expects two.